event-sourcing/subscription-manager/src/main/java/uk/gov/justice/services/event/sourcing/subscription/SubscriptionManagerProducer.java

```diff
@@ -25,23 +35,31 @@
     @Inject
     SubscriptionDescriptorDefinitionRegistry subscriptionDescriptorRegistry;
 
-    @Inject
-    InterceptorChainProcessorProducer interceptorChainProcessorProducer;
-
     @Inject
     QualifierAnnotationExtractor qualifierAnnotationExtractor;
 
     @Produces
     @SubscriptionName
     public SubscriptionManager subscriptionManager(final InjectionPoint injectionPoint) {
-
         final SubscriptionName subscriptionName = qualifierAnnotationExtractor.getFrom(injectionPoint, SubscriptionName.class);
+        LOGGER.info(format("Creating subscription manager for subscription name: %s", subscriptionName.value()));
```

Comment:
This should be at debug level at most, and the message is no longer correct since refactoring

Reply:
Done but not sure why this message is not right